packages/components/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 ### Internal
 
 -   `Mobile` Refactor of the KeyboardAwareFlatList component.
+-   `BottomSheetCell`: Refactor away from Lodash (mobile) ([#49794](https://github.com/WordPress/gutenberg/pull/49794)).
+-   `parseStylesVariables()`: Refactor away from Lodash (mobile) ([#49794](https://github.com/WordPress/gutenberg/pull/49794)).
+-   Remove Lodash dependency from components package ([#49794](https://github.com/WordPress/gutenberg/pull/49794)).
```

Comment:
We're adding 3 separate entries since we're touching 2 distinct components and removing a dependency.